src/js/components/awardv2/idv/ComingSoonSection.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import InfoTooltip from './InfoTooltip';
+
+const propTypes = {
+    title: PropTypes.string,
+    icon: PropTypes.string,
+    className: PropTypes.string,
+    toolTip: PropTypes.bool,
+    toolTipAlignLeft: PropTypes.bool,
+    includeHeader: PropTypes.bool,
+    toolTipContent: PropTypes.node,
+    children: PropTypes.node
+};
+
+const defaultProps = {
+    toolTip: false,
+    includeHeader: false,
+    className: null
+};
+
+const ComingSoonSection = ({
+    title,
+    icon,
+    toolTip,
+    toolTipAlignLeft,
+    toolTipContent,
+    includeHeader,
+    children,
+    className
+}) => {
+    const renderChildren = () => (
+        children || (
```

Comment:
Like that this component is very re-usable!

Reply:
💪